content/en/_index.md

```diff
@@ -32,12 +32,12 @@ Mastodon uses a standardized, open protocol to implement federation. It is calle
 
 The **fediverse** (“federated universe”) is the name for all websites that can communicate with each other over ActivityPub and the World Wide Web. That includes all Mastodon servers, but also other implementations:
 
-* Pleroma, a modular microblogging engine,
-* Pixelfed, federated image sharing platform, which lets you share and consume media posts,
-* Misskey, which includes microblogging alongside a customizable dashboard,
-* PeerTube, which lets you upload videos to channels,
-* Plume, which lets you publish longer-form articles,
-* and many more, including individual and personal websites!
+* Pleroma, a modular microblogging engine;
```

Comment:
Do we really need line ending punctuation here at all? Seems inconsistent with style of other areas. Also let's revise the `*` to `-` such as in other areas.

Reply:
I think we need a punctuation (particularly a semicolon) at the end of each line to separate each item sir.
It’s a list of long sentences which follow on from the first item down to the last.
The semicolon makes it easier to read them as separate items or sentences in the same list.